src/extension/module.js

```diff
@@ -2865,6 +2901,11 @@
       // toggle sidekick
       window.hlx.sidekick.toggle();
     }
+    if(window.hlx.sidekick.isEditor()){
+      sampleRUM('sidekick:open', {
+             source: '', // TODO: the current URL being edited or previewed (this can be the word or google docs URL)
+          });
+    }
```

Comment:
Why do we differentiate between editor or not here? Also, technically it doesn't have to be "open" at this point (can also stay be collapsed, based on the user's preference).

If you want to track open/close, I'd move the `sampleRUM` call to the respective methods `show` and `hide`. However, we would have to take into account, that the sidekick is often shown automatically as long as the user doesn't explicitly close it.